### PR TITLE
Add coverage report (+ slightly improved the coverage)

### PR DESCRIPTION
**Pull request by lbeder**

### CODE REVIEW

File: test/network/NetworkSettings.ts [hunk 1]
```diff
@@ -179,13 +181,21 @@ describe('NetworkSettings', () => {
                 const reserveToken2 = await createTestToken();
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.false;
                 expect(await networkSettings.isTokenWhitelisted(reserveToken2.address)).to.be.false;
+                expect(await networkSettings.protectedTokenWhitelist()).not.to.have.members([
```
Comment (barakman) [inline test/network/NetworkSettings.ts:184]: `to.be.empty` would be a little more restrictive.

File: test/network/NetworkSettings.ts [hunk 2]
```diff
@@ -179,13 +181,21 @@ describe('NetworkSettings', () => {
...
+                    reserveToken2.address
+                ]);
 
                 const res = await networkSettings.addTokensToWhitelist([reserveToken.address, reserveToken2.address]);
                 await expect(res).to.emit(networkSettings, 'TokenAddedToWhitelist').withArgs(reserveToken.address);
                 await expect(res).to.emit(networkSettings, 'TokenAddedToWhitelist').withArgs(reserveToken2.address);
 
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.true;
                 expect(await networkSettings.isTokenWhitelisted(reserveToken2.address)).to.be.true;
+                expect(await networkSettings.protectedTokenWhitelist()).to.have.members([
```
Comment (barakman) [inline test/network/NetworkSettings.ts:195]: `to.deep.equal` would be a little more restrictive.

File: test/network/NetworkSettings.ts [hunk 3]
```diff
@@ -211,11 +221,13 @@ describe('NetworkSettings', () => {
 
             it('should remove a token', async () => {
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.true;
+                expect(await networkSettings.protectedTokenWhitelist()).to.include(reserveToken.address);
 
                 const res = await networkSettings.removeTokenFromWhitelist(reserveToken.address);
                 await expect(res).to.emit(networkSettings, 'TokenRemovedFromWhitelist').withArgs(reserveToken.address);
 
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.false;
+                expect(await networkSettings.protectedTokenWhitelist()).not.to.include(reserveToken.address);
```
Comment (barakman) [inline test/network/NetworkSettings.ts:230]: Semantic: I'd use `to.not.include` (3 places in this file, one of which is not part of this PR).

**Review by barakman — APPROVED** [review]

File: test/network/NetworkSettings.ts (hunk 1 above)
Comment (lbeder) [reply test/network/NetworkSettings.ts:184]: The test only checks the difference between adding/removing specific tokens, so it's unnecessary also to assume that the list is empty (which is incidentally the case)

File: test/network/NetworkSettings.ts (hunk 2 above)
Comment (lbeder) [reply test/network/NetworkSettings.ts:195]: The test only checks the difference between adding/removing specific tokens, so it's unnecessary also to assume that the list was empty (which is incidentally the case)

File: test/network/NetworkSettings.ts (hunk 3 above)
Comment (lbeder) [reply test/network/NetworkSettings.ts:230]: Sure

Comment (lbeder) [reply test/network/NetworkSettings.ts:230]: Actually, it works either way, so let's not waste time on this
